src/screens/WalletConnect/WalletConnectList.js

```diff
@@ -67,13 +68,13 @@ const style = StyleSheet.create({
   },
   sessionName: {
     marginBottom: 8,
-    color: 'rgba(0, 0, 0, 0.87)',
+    color: COLORS.textColorShadowDark,
```

Comment:
Changed from `0.87` opacity to `0.8` opacity

Reply:
Do we have a constant with `0.9`?

Reply:
Reverted to `0.9` opacity on [a6b16fd](https://github.com/HathorNetwork/hathor-wallet-mobile/pull/327/commits/a6b16fdf489a4652038ac614da23fe5485e62020)